codes/model-net.h

```diff
@@ -79,6 +86,7 @@ typedef struct mn_stats mn_stats;
     X(DRAGONFLY_PLUS_ROUTER, "modelnet_dragonfly_plus_router", "dragonfly_plus_router", &dragonfly_plus_router_method)\
     X(DRAGONFLY_DALLY, "modelnet_dragonfly_dally", "dragonfly_dally", &dragonfly_dally_method)\
     X(DRAGONFLY_DALLY_ROUTER, "modelnet_dragonfly_dally_router", "dragonfly_dally_router", &dragonfly_dally_router_method)\
+    X(CONGESTION_CONTROLLER, "congestion_controller", "congestion_controller", NULL)\
```

Comment:
I'm pretty certain that the current version of Congestion Control doesn't require this but I need to double check

Reply:
This is actually necessary because Congestion Controller methods inside of router LPs send congestion controller messages which are a different size than the network.